python-threatexchange/threatexchange/exchanges/clients/techagainstterrorism/tests/test_api.py

```diff
@@ -2,31 +2,37 @@
 
 import typing as t
 import pytest
-from datetime import datetime
 
 from threatexchange.exchanges.clients.techagainstterrorism.api import (
     TATIdeology,
-    TATHashListResponse,
     TATHashListAPI,
 )
 
 
 def mock_get_hash_list(
     ideology: str = TATIdeology._all.value,
-) -> t.Union[TATHashListResponse, t.Dict[str, str]]:
-
-    if ideology not in TATIdeology._value2member_map_:
-        return {
-            "error": f"400 Client Error: Bad Request for url: https://test/api/hash-list/{ideology}"
-        }
-
-    return TATHashListResponse(
-        file_url=f"https://hash-list.s3.com/19700101_{ideology}_hashes.json",
-        file_name=f"19790101_{ideology}_hashes.json",
-        created_on=datetime.strptime("2021-08-01T00:00:00Z", "%Y-%m-%dT%H:%M:%SZ"),
-        total_hashes=1000,
-        ideology=ideology,
-    )
+) -> t.Optional[t.List[t.Dict[str, str]]]:
+
+    return [
+        {
+            "hash_digest": "12345abcde",
+            "algorithim": "MD5",
+            "ideology": ideology,
+            "file_type": "jpg",
+        },
+        {
+            "hash_digest": "12345abcde",
+            "algorithim": "MD5",
+            "ideology": ideology,
+            "file_type": "jpg",
+        },
+        {
+            "hash_digest": "12345abcde",
+            "algorithim": "MD5",
+            "ideology": ideology,
+            "file_type": "jpg",
+        },
```

Comment:
ignorable: I can't tell whether ot not these are changing between cases, you can could put this in a global to reference multiple times during the test